### ISSUE TITLE
Immutant has been broken since Java 1.09+

### ISSUE THREAD

**Issue by WorldsEndless** [luminus-framework/luminus#293]
It has been our team's experience that WildFly deployment using immutant has been broken since Java  11, c/o https://issues.redhat.com/browse/IMMUTANT-645 , and immutant itself has been deprecated and undeveloped since 2018. I tried to pick up the pieces and make some fixes to the deps, but Immutant was a dauntingly large project despite being open source, and I never got traction. 

If there is a different way to deploy to WildFly, I would love to know it! Either way, the documentation may need fixing. I'd be happy to make those fixes a PR on the documentation repo, but would really love to hear a different WildFly solution if possible.  

https://luminusweb.com/docs/deployment.html#deploying_to_wildfly

**Comment by yogthos** [comment]
Yeah, there's [ring-undertow-adapter](https://github.com/luminus-framework/ring-undertow-adapter) for standalone deployments now, but nothing's been done for deploying to WildFly as a war at the moment. The new adapter incorporates some of the code from Immutant but it's much leaner, so it might be easier to extend or build on top of for war deployment.

I haven't used app servers for deployments in a while as I tend to run apps as uberjars, so unfortunately not too familiar with what would be involved there. If you do get a solution, a PR would be highly appreciated. :)
